osc-server/src/main/java/org/osc/core/broker/service/validator/DeleteDistributedApplianceRequestValidator.java

```diff
@@ -53,6 +60,10 @@ public DistributedAppliance validateAndLoad(BaseDeleteRequest request) throws Ex
                             da.getId()));
         }
 
+        // DE5296->Error message to be more appropriate when deleting a DA without
+        // deleting the SFC
```

Comment:
Does not look like the line below is deleting, pls remote this comment.

Reply:
Thank you for your review comments .I made changes